changelogs/fragments/62007-add-domain-module.yml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - ecs_domain - Addition of ``ecs_domain`` module (https://github.com/ansible/ansible/pull/60883).
```

Comment:
New modules don't need a changelog fragment. (It is automatically generated by tooling.)